File: HW01/CPP/src/VolleyballNet.cpp
#include "../include/VolleyballNet.h"
#include <iostream>
#include <string>
using namespace std;

VolleyballNet::VolleyballNet(string type, string size, double price) : IndoorItem(price) {
    this->type = type;
    this->size = size;
    this->price = price;
}

void VolleyballNet::display() {
    cout << "$" << this->price << " - This volleyball net is made for " << this->type << "s and is " << this->size;
}

File: HW01/PYTHON/project/VolleyballNet.py
from .IndoorItem import IndoorItem

class VolleyballNet(IndoorItem):
    def __init__(self, price, type, size):
        super().__init__(price)
        self.type = type
        self.size = size
    
    def display(self):
        print(f"${self.price} - This volleyball net is made for {self.type}s and is {self.size}", end="")